this include is not needed, and we can't use it on PyPy because ctx_type.h
includes <Python.h>. Fixed upstream by PR 201

diff --git a/pypy/module/_hpy_universal/_vendored/hpy/devel/src/runtime/ctx_tracker.c b/pypy/module/_hpy_universal/_vendored/hpy/devel/src/runtime/ctx_tracker.c
--- a/pypy/module/_hpy_universal/_vendored/hpy/devel/src/runtime/ctx_tracker.c
+++ b/pypy/module/_hpy_universal/_vendored/hpy/devel/src/runtime/ctx_tracker.c
@@ -58,7 +58,7 @@
  */
 
 #include "hpy.h"
-#include "common/runtime/ctx_type.h"
+//#include "common/runtime/ctx_type.h"
 
 static const HPy_ssize_t HPYTRACKER_INITIAL_CAPACITY = 5;
 
